README.md

```diff
@@ -78,6 +79,21 @@ To launch the interactive CLI, run the executable without any arguments.
 
 Run `ficsit help` to see a list of available commands.
 
+## Game Installations
+
+The app needs to know where the game is installed. On Steam, this will be
+
+    C:\Program Files (x86)\Steam\steamapps\common\Satisfactory
+
+For MacOS and Crossover use (Assuming your bottle is named "Steam"):
+
+    ~/Library/Application\ Support/CrossOver/Bottles/Steam/drive_c/Program\ Files\ \(x86\)/Steam/steamapps/common/Satisfactory
+
+The command to add an installation is then:
```

Comment:
```suggestion
To add an installation, either use the interactive interface, or use the command:
```

Reply:
Superseded by https://github.com/satisfactorymodding/ficsit-cli/pull/61/files#r1562711449

Reply:
> The interactive interface doesn't actually have a way to add an installation.

It does, through the manage installs screen, unless it has changed since I last worked on this project